doc/authors.md

```diff
@@ -1,36 +1,45 @@
 Zcash Contributors
 ==================
 
-Jack Grigg (323)
-Simon Liu (220)
-Sean Bowe (180)
+isle2983 (2983)
```

Comment:
One commit from that user has been backported.  So this is probably a bug in contributions script.

> commit 5dfea488e4c672f05971eff6da1bfce863f83b6c
> Author: isle2983 <isle2983@yahoo.com>
> Date:   Sun Sep 11 15:26:53 2016 -0600
> 
>     [copyright] add MIT License copyright header to zmq_sub.py
>